samples/amber.cc

```diff
@@ -108,7 +116,7 @@ bool ParseArgs(const std::vector<std::string>& args, Options* opts) {
       std::cerr << "Unrecognized option " << arg << std::endl;
       return false;
     } else {
-      opts->input_filename = args[i];
+      opts->input_filenames.push_back(args[i]);
```

Comment:
This should verify the filename is not empty otherwise I could do `amber "" "" ""`